public/regdesk.html

```diff
@@ -55,9 +55,15 @@
     </nav>
     <div class="row row-no-margin container-no-gutter h-100">
       <div class="col-2 d-grid gap-2 h-100 container-no-gutter-left">
-        <button id="cancelBtn" class="btn btn-danger btn-fullheight btn-huge">CANCEL</button>
+        <button id="cancelBtn" class="btn btn-danger btn-fullheight btn-huge invisible">CANCEL</button>
       </div>
-      <div class="col-8 pt-3" id="centerField">
+      <div class="col-8 pt-3 h-100 overflow-auto" id="centerField">
+        <div id="hideAfterPageLoad" class="row">
+          <div class="alert alert-warning" role="alert">
+            <h2>Search bar not loading? Try refreshing the page.</h2>
```

Comment:
Fullscreen app, won't have a refresh button :P

Reply:
All Chromebooks come with a dedicated refresh button to replace F5 :D